### DISCUSSION TITLE
本地开发登录401

### DISCUSSION THREAD

**Discussion by so-colorful** [nezhahq/skeleton-custom-theme#4] [General]
提示错误信息：oauth2: cannot fetch token: 401 Unauthorized Response: {"error":"invalid_grant","error_description":"授权方式无效，或者登录回调地址无效、过期或已被撤销"}
![image](https://user-images.githubusercontent.com/35517429/191547992-471e5554-7919-434d-b6b2-9e8d072b6103.png)
![image](https://user-images.githubusercontent.com/35517429/191548037-4813b2db-a644-4872-9d04-863a88ebb4ff.png)

**Comment by naiba** [comment]
你不是 clientId 和 client secret 复制重了
